js/renovation/ui/scroll_view/__tests__/scrollable.test.tsx

```diff
@@ -992,44 +1001,130 @@ each([{
+                              each([-100, 0]).describe('Scrollbar position: %o', (scrollbarPosition) => {
+                                it('validate method in simulated strategy', () => {
+                                  if (Scrollable === ScrollableNative) {
+                                    return; // this config is not relevant to the native strategy
+                                  }
+
+                                  const viewModel = new Scrollable({
+                                    direction, bounceEnabled, disabled, scrollByContent,
+                                  }) as any;
+
+                                  initRefs(viewModel);
+
+                                  initStyles((viewModel).containerRef, containerSize);
+                                  initStyles((viewModel).contentRef, contentSize);
+
+                                  setScrollbarPosition(viewModel.horizontalScrollbarRef,
+                                    scrollbarPosition);
+                                  setScrollbarPosition(viewModel.verticalScrollbarRef,
+                                    scrollbarPosition);
+
+                                  viewModel.cachedVariables.locked = locked;
+
+                                  let expectedValidationResult;
+                                  if (disabled || locked) {
+                                    expectedValidationResult = false;
+                                  } else if (bounceEnabled) {
+                                    expectedValidationResult = true;
+                                  } else if (isDxWheelEvent) {
+                                    expectedValidationResult = (contentSize > containerSize)
+                                      && (
+                                        (scrollbarPosition < 0 && delta > 0)
+                                        || (scrollbarPosition >= 0 && delta < 0)
+                                      );
+                                  } else if (!scrollByContent && !isScrollbarClicked) {
+                                    expectedValidationResult = false;
+                                  } else {
+                                    expectedValidationResult = containerSize < contentSize
+                                      || bounceEnabled;
+                                  }
+
+                                  const target = isScrollbarClicked
+                                    ? viewModel.containerRef.querySelector(`.${SCROLLABLE_SCROLLBAR_CLASS}`)
+                                    : viewModel.containerRef;
+                                  const e = { ...defaultEvent, target, delta };
+                                  if (isDxWheelEvent) {
+                                    (e as any).type = 'dxmousewheel';
+                                  }
+
+                                  const actualResult = (viewModel).validate(e);
+                                  viewModel.disposeWheelTimer()();
```

Comment:
We need to check that wheel timer was cleared

Reply:
Yep, great idea.
Also checking how this timer affects the final result